### PR TITLE
Wyre widget android

### PR DESCRIPTION
**Pull request by brunobar79**
This re-enables all Apple Pay features for Android but defaults to wyre checkout widget through a webview instead of showing the Add Cash sheet.

DEMO: http://recordit.co/Ifgq4vEupX

@christianbaroni  Waiting for your feedback on how to make the navbar look better

### CODE REVIEW

**Comment by brunobar79** [comment]
BTW, I wasn't able to fully test this because I don't know what credentials to use on dev mode, and the app crashes for me in release mode  (unrelated to this)  with a real debit card before I can finish the payment flow.

@jin do you have any idea about test creds for wyre?

**Comment by osdnk** [comment]
I think we should do sth to make network toast not visible outside the main screen, like @skylarbarrera said

File: android/app/src/main/java/me/rainbow/MainActivity.java [hunk 1]
```diff
@@ -16,6 +17,7 @@
   protected void onCreate(Bundle savedInstanceState) {
         SplashScreen.show(this, R.style.NoActionBar);
         super.onCreate(savedInstanceState);
+        WebView.setWebContentsDebuggingEnabled(true);
```
Comment (osdnk) [inline android/app/src/main/java/me/rainbow/MainActivity.java:20]: what's this?

**Review by osdnk — APPROVED** [review]
looks meh, but better than nothing

File: android/app/src/main/java/me/rainbow/MainActivity.java (hunk 1 above)
Comment (brunobar79) [reply android/app/src/main/java/me/rainbow/MainActivity.java:20]: To be able to debug the webview from chrome. Doesn't change anything for end users, but it's useful for us.

**Review by skylarbarrera — APPROVED** [review]
LGTM, 100% need to hide testnet toasts across the board except for main nav before beta

**Comment by brunobar79** [comment]
I agree with both of you but the toast fix is unrelated to this PR, so let's get this merged and fix that later.

**Pull request merged**